recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -0,0 +1,162 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,14 +1,14 @@
+-project(basisu)
```

Comment:
As a general note, when you see a poor cmake script that needs to be patched (heavily).
It's always nice to also send the changes to the project itself. That way, upstream benefits too from your work.